examples/plugins/example-plugin/module/vtk.module

```diff
@@ -5,3 +5,4 @@ DESCRIPTION
 DEPENDS
   VTK::CommonCore
   VTK::CommonExecutionModel
+  VTK::IOImport
```

Comment:
I know this is needed because reader.h expose a vtkImporter, maybe we should use raw pointers and forward declare it. Not essential though so feel free to merge

Reply:
Yes, currently needed, but of course can be improved if you feel like it.